### PR TITLE
Krypton: Keep Autotools working

### PR DESCRIPTION
**Pull request by fritsch**
This keeps autotools alive.

### CODE REVIEW

**Comment by fritsch** [comment]
jenkins build this please (just want to see if it still does what it should)

File: xbmc/cores/AudioEngine/AEDefines.h [hunk 1]
```diff
@@ -22,5 +22,7 @@
 #define AE_AC3_ENCODE_BITRATE 640000
 #define AE_DTS_ENCODE_BITRATE 1411200
 
+#ifdef BUILDING_WITH_CMAKE
```
Comment (FernetMenta) [inline xbmc/cores/AudioEngine/AEDefines.h]: what is this, still an ugly ifdef hack

Comment (fritsch) [inline xbmc/cores/AudioEngine/AEDefines.h]: It's a cmake define, like the one for SSE the one for X11, used e.g. here: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp#L49 the one for MIR, the one for whatever ... can you explain me the difference between your problem with the ifdef in this place and the ifdef that you pushed some weeks ago into VAAPI?

I think I have a problem calibrating towards your wishes.

**Comment by fritsch** [comment]
Oki - next try on the way. Calibration ongoing ...

File: xbmc/cores/AudioEngine/AEDefines.h (hunk 1 above)
Comment (FernetMenta) [inline xbmc/cores/AudioEngine/AEDefines.h]: put it in words: why should the code know if its built with cmake or whatever system else?

Comment (stefansaraev) [inline xbmc/cores/AudioEngine/AEDefines.h]: because it is a header, generated only by the cmake buildsystem ?

**Pull request closed without merging**